explorer/explorer.go

```diff
@@ -414,34 +337,28 @@ func (exp *explorerUI) addRoutes() {
 	exp.Mux.Use(middleware.Recoverer)
 	corsMW := cors.Default()
 	exp.Mux.Use(corsMW.Handler)
-	//Basically following the same format as the apiroutes
+
 	exp.Mux.Get("/", exp.root)
 
 	exp.Mux.Route("/block", func(r chi.Router) {
-		r.Route("/best", func(rd chi.Router) {
-			rd.Use(exp.app.BlockHashLatestCtx)
-			rd.Get("/", exp.blockPage)
-		})
-
 		r.Route("/{blockhash}", func(rd chi.Router) {
-			rd.Use(exp.app.BlockHashPathOrIndexCtx)
+			rd.Use(exp.blockHashPathOrIndexCtx)
```

Comment:
It looked like the blockPage is only using `getBlockHashCtx` now, so can't this middleware be a simpler handler that does not try to get the height when it's not needed?  Or or is this route intended to handle height's input here instead of a hash?

Reply:
It handles both hash and height so http://localhost:7777/explorer/block/174288 and http://localhost:7777/explorer/block/000000000000000d1653f66e60c269e7fcbace5841005624ae47d680561d1b14 link to the same page

Reply:
OIC.